### ISSUE TITLE
Training data annotation bbox 

### ISSUE THREAD

**Issue by abhigoku10** [ghimiredhikura/Complex-YOLO#1]
@ghimiredhikura   Hi thanks for  a wonderful code it was really helpful , I have few queries mentioned below 
1. When is draw annotation box on BEV map in few images i am not able to get the annotation of the bounding box , did u also face this issue 
2. Did u train the model if so can you share the trained model and let me the accuracy 
3. Where you able to get the correct label in you predication code 
4.Where you able to draw 3 bounding box on a 2D image 
Thanks in advance

**Comment by ghimiredhikura** [comment]
hello @abhigoku10

> @ghimiredhikura Hi thanks for a wonderful code it was really helpful , I have few queries mentioned below
> 
> 1. When is draw annotation box on BEV map in few images i am not able to get the annotation of the bounding box , did u also face this issue

Sorry, couldn't understand your problem, you mean gt bounding box? 

> 2. Did u train the model if so can you share the trained model and let me the accuracy

I am still working on this, therefore I haven't calculated training accuracy yet. 

> 3. Where you able to get the correct label in you predication code

There are several errors and missing predictions. Class prediction is not working well. 

>    4.Where you able to draw 3 bounding box on a 2D image

Later I will add this feature. 

>    Thanks in advance

**Comment by abhigoku10** [comment]
@ghimiredhikura Thanks for the response . 
The first point is , i am trying to draw the bounding box on the BEV map with the data from label but in few images i am not able to get any bounding box draw since the this condition "**if  (location_x>0) & (location_x<40) & (location_y>-40)  & (location_y<40) :**" is not satisifed for example for 00001.png image from kitti dataset . Where you also facing the same issue

**Comment by ghimiredhikura** [comment]
Yes, this is because we are drawing bounding box only on the front view of lidar cloud, i.e. same view as corresponding 2D image.

**Comment by abhigoku10** [comment]
@ghimiredhikura  for few images its missing partially also due to which i most of the images i am having no bounding box on the BEV feature

**Comment by abhigoku10** [comment]
@ghimiredhikura  in the current training code the validation loss and the validation set is not considered during training

**Comment by ghimiredhikura** [comment]
I will add this part in next commit. Can you do the training with latest commit code and let me know results. Lets reproduce the result in paper together.

**Comment by abhigoku10** [comment]
@ghimiredhikura  sure i can take your latest code and do the training and update the results. When is your next comit. with validation loss ??

**Comment by ghimiredhikura** [comment]
@abhigoku10, I want to make sure training is working well, after that I will add validation accuracy and loss stuffs.

**Comment by abhigoku10** [comment]
@ghimiredhikura  sure give me a couple of days time i shall do it . can you share your mail ID or means where i can talk to you

**Comment by ghimiredhikura** [comment]
@abhigoku10, sure, ghmdeepak@gmail.com
